README.md

```diff
@@ -107,19 +108,27 @@ pip install -e ".[dev]"
 ```
 and follow the more detailed instructions in [CONTRIBUTING](https://github.com/facebookresearch/theseus/blob/main/CONTRIBUTING.md).
 
-Compilation from source supports is affected by the following enviroment variables:
-* `BASPACHO_ROOT_DIR`: root dir of [BaSpaCho](https://github.com/facebookresearch/baspacho) sparse solver, which must be already
-  built with binaries in the subdirectory `build`.
-* `THESEUS_FORCE_CUDA` (1 or 0): if set, forces enabling or disabling Cuda support regardless of system's `torch.cuda.is_available()`.
+**Installing BaSpaCho extensions from source**
```

Comment:
For readability I would suggest making the following inner headings as bullets

- #### **pypi**
- #### **From source**
- #### **Installing BaSpaCho extensions from source**